### PR TITLE
Add Powerstore v2.6 samples

### PR DESCRIPTION
**Pull request by panigs7**
# Description
- Added v2.6 CSI Powerstore driver support
- Changes to update storage capacity value

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/613 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

Sanity test:
image
image

### CODE REVIEW

File: operatorconfig/driverconfig/powerstore/v2.6.0/controller.yaml [hunk 1]
```diff
@@ -0,0 +1,255 @@
+apiVersion: v1
```
Comment (rajendraindukuri) [inline operatorconfig/driverconfig/powerstore/v2.6.0/controller.yaml:1]: @panigs7 Copyrights are missing

File: operatorconfig/driverconfig/powerstore/v2.6.0/node.yaml [hunk 2]
```diff
@@ -0,0 +1,214 @@
+apiVersion: v1
```
Comment (rajendraindukuri) [inline operatorconfig/driverconfig/powerstore/v2.6.0/node.yaml:1]: @panigs7 Copy rights are missing

File: operatorconfig/driverconfig/powerstore/v2.6.0/controller.yaml (hunk 1 above)
Comment (panigs7) [reply operatorconfig/driverconfig/powerstore/v2.6.0/controller.yaml:16]: Done

**Review by rensyct — DISMISSED** [review]

File: samples/storage_csm_powerstore_v250.yaml [hunk 3]
```diff
@@ -27,6 +27,7 @@ spec:
       # Allowed values: ReadWriteOnceWithFSType, File , None
       # Default value: ReadWriteOnceWithFSType
       fSGroupPolicy: "ReadWriteOnceWithFSType"
+      storageCapacity: true
```
Comment (karthikk92) [inline samples/storage_csm_powerstore_v250.yaml:30]: Please describe the of variable storageCapacity ref: fSGroupPolicy

**Review by rajendraindukuri — APPROVED** [review]

**Review by karthikk92 — APPROVED** [review]

**Review by rajendraindukuri — APPROVED** [review]

File: samples/storage_csm_powerstore_v250.yaml (hunk 3 above)
Comment (panigs7) [reply samples/storage_csm_powerstore_v250.yaml]: Done

**Review by rensyct — APPROVED** [review]

**Pull request merged**